src/common/i18n/messages.ts

```diff
@@ -177,7 +177,9 @@ export const BASE_LOCALE = {
   'ld.from': 'From',
   'ld.to': 'To',
   'ld.apply': 'Apply',
-  'ld.notSpecified': 'Not specified'
+  'ld.notSpecified': 'Not specified',
+  'ld.externalResource': 'External resource',
+  'ld.fetchingExternalResourceById': 'Fetching external resource id {resourceId}...'
```

Comment:
Fetching external resource **by** id {resourceId}...?

Reply:
This preposition is not necessary here. Here's a structurally similar sentence: "Fetching order number 3"